src/main/java/net/runelite/rsb/wrappers/subwrap/NPCChat.java

```diff
@@ -17,10 +21,32 @@ public String getName() {
     }
 
     public String[] getOptions() {
-        return new String[]{};
+        try {
+            return Arrays.stream(getComponent(GlobalWidgetInfo.DIALOG_DYNAMIC_CONTAINER).getComponents()).map(RSWidget::getText).toArray(String[]::new);
+        } catch (Exception e) {
+            return null;
+        }
+    }
+
+    public boolean containsOption(String option){
+        try {
+            return Arrays.stream(getComponent(GlobalWidgetInfo.DIALOG_DYNAMIC_CONTAINER).getComponents()).anyMatch(o->o.getText().contains(option));
+        } catch (Exception e) {
+            return false;
+        }
     }
 
     public boolean selectOption(String option, boolean wait) {
+
+        final RSWidget iface = getComponent(GlobalWidgetInfo.DIALOG_DYNAMIC_CONTAINER);
+        if(iface.getComponents()!=null){
```

Comment:
some better var naming, what is exactly a iface? a  ith face of someting? a dialog interface?

```java
final RSWidget iface = getComponent(GlobalWidgetInfo.DIALOG_DYNAMIC_CONTAINER);
```
fix spaces pls

```java
if (iface.getComponents()!=null) {
```

Reply:
iface is a result of last-min "just make this work".  will fix.  :)